### ISSUE TITLE
Amazon introduced a captcha to accept offers in its latest update.

### ISSUE THREAD

**Issue by gmcsnetwork** [mdesilva/AmazonFlexUnlimited#92]
Anyone else noticed this on the latest Amazon Flex app update? Getting report from colleagues that now at random they have to complete a captcha before accepting an offer. This prevents the script from accepting offers until after the user manually accepts one offer and completes the captcha.

**Comment by kyleanderson84** [comment]
Yes, response code 307 is what I saw, was able to go in to the app and complete it, now I'm getting response code 410 again for missed blocks. I'm going to add a bit of code to send a text to prompt me to log in to the app and solve the puzzle. I think it should pause the script until I manually tell it to start again also.

**Comment by gmcsnetwork** [comment]
Thanks for getting back with the error code, that sounds like a decent workaround.

**Comment by gmcsnetwork** [comment]
Can someone share a print screen of the reCAPTCHA or captcha from the Amazon Flex app? I have not seen it myself yet, and I am wondering how complex it is. There are a few scripts that could potentially help to get around it.

https://github.com/topics/recaptcha-solver-python

**Comment by kyleanderson84** [comment]
![FB_IMG_1679526836463.jpg](https://user-images.githubusercontent.com/29053423/227059277-aea1b95d-3d4f-4f5b-b7a4-6f2cd965c7f7.jpg)

**Comment by gmcsnetwork** [comment]
Thank you for sharing the print screen, how frequently are you getting the prompted to solve the captcha? if it's for every other offer it will make using the script imposible...

**Comment by kyleanderson84** [comment]
I don't know what triggers it, but I've only gotten it once and have picked up two routes since they started, so it's not very frequent

**Comment by rgribeiro2009** [comment]
> Sim, o código de resposta 307 é o que eu vi, consegui entrar no aplicativo e concluí-lo, agora estou recebendo o código de resposta 410 novamente para blocos perdidos. Vou adicionar um pouco de código para enviar um texto para me solicitar o login no aplicativo e resolver o quebra-cabeça. Eu acho que deve pausar o script até que eu diga manualmente para começar de novo também.

Hey there! I see you're planning some code changes. Mind sharing the code snippet you're gonna implement with the fix you have in mind? That'd be awesome and really help me understand the changes better. Thanks
